blog/2021-04-27-sse.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "What's SSE and how to use it"
+author: "xxchan"
+author_url: "https://github.com/xxchan"
+tags: [updates]
+---
+
+Data protection includes protecting data while in transit (as it travels to and from the server) and at rest (while it is stored on disk). You can do the following to protect your data when using storage services:
+
+- **Secure Socket Layer/Transport Layer Security (SSL/TLS)**: Protect data in transit.
+- **Client-Side Encryption**: You manage the encryption keys, encryption process, and related tools all yourself. This can protect data both in transit and at rest, and this is transparent to the server.
+- **Server-Side Encryption**: Request the server to encrypt your data before it is persisted to disk. 
+
+Recently, we [implemented SSE support](https://github.com/aos-dev/go-storage/issues/523) in [go-storage], and let's talk about SSE here.
```

Comment:
How about adding a notice for abbr `SSE` like `about Server-Side Encryption (shorten to SSE) here`?